### PR TITLE
Extend firstaid for cleveref for gh1560

### PR DESCRIPTION
**Pull request by u-fischer**
# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [ ] Rollback provided (if necessary)?
- [ ] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: required/firstaid/latex2e-first-aid-for-external-files.dtx [hunk 1]
```diff
@@ -740,12 +740,16 @@
 %    \end{macrocode}
 %  we test if \cs{@currentcounter} is empty for unnumbered sections
 % \changes{v1.1h}{2024/11/01}{fix spurious space}
+% \changes{v1.1j}{2024/11/24}{add \cs{iftag@} test, issue \#1560}
 %    \begin{macrocode}
+       \newif\iftag@
```
Comment (FrankMittelbach) [inline required/firstaid/latex2e-first-aid-for-external-files.dtx:745]: I'm surprised that \newif doesn't bark, when the conditional aready exists, but it looks like that in fact the case. Not sure how god that is, but then it must have been that way forever.

File: required/firstaid/changes.txt [hunk 2]
```diff
@@ -1,3 +1,7 @@
+2024-11-24  Ulrike Fischer  <Ulrike.Fischer@latex-project.org>
+
+	* latex2e-first-aid-for-external-files.dtx: add iftag@-test to cleveref firstaid for issue #1560
```
Comment (FrankMittelbach) [inline required/firstaid/changes.txt:3]: add \iftag@ test ...

(but not important)

**Review by FrankMittelbach — APPROVED** [review]
modulo the fact that I think this needs to be based on main for the hotfix, I think this is fine.

File: required/firstaid/latex2e-first-aid-for-external-files.dtx (hunk 1 above)
Comment (josephwright) [reply required/firstaid/latex2e-first-aid-for-external-files.dtx:745]: Yes, it comes from plain and plain doesn't check anything :)

**Review by josephwright — APPROVED** [review]

**Pull request merged**